.git-blame-ignore-revs

```diff
@@ -0,0 +1,7 @@
+# Ignore calls to `cargo fmt`
+#
+# To learn more, see
+#     https://docs.github.com/en/repositories/working-with-files/using-files/viewing-a-file#ignore-commits-in-the-blame-view
+
+# Cargo fmt changing max width to 80
+94b7757698baa135a2f2139f3669af10f7e71e60
```

Comment:
I'm guessing you're still working through this. If I'm not mistaken, I currently don't see this commit in this PR? Also, I recommend:
* Commit 1: Make config change
* Commit 2: Reformat the code base
* Commit 3: Introduce the ignore-revs file with commit 2

Reply:
Yeah, I think I messed up a bit when copying it.

Done now though.